docs/dev_guide/migration/debug-with-codepin.rst

```diff
@@ -0,0 +1,224 @@
+CodePin
+===============
+
+There are cases that the migrated SYCL program has different runtime behavior to
+the original CUDA program. The reason of the inconsistency could be:
+* Difference in arithmetic precision between hardware.
+* Semantic difference between the CUDA and SYCL APIs
+* Errors introduced during the automatic migration
+
+CodePin is introduced as a sub-feature of |tool_name| to reduce the effort of
```

Comment:
change to "in order to reduce the effort of debugging such inconsistencies in runtime behavior."